lib/github.com/diku-dk/containers/setops.fut

```diff
@@ -0,0 +1,141 @@
+import "../sorts/radix_sort"
+import "opt"
+
+-- | Set-like operations on arrays of values interpreted as sets
+
+local
+-- | The setops module type.  This module type is declared `local`, which means
+-- that it may not be referenced directly by name from client code.  This
+-- limitation makes it possible for the interface to be enriched by new members
+-- in future minor versions.
+
+module type setops = {
+  -- | [union a b] returns the set of elements that appear in both a and b with
```

Comment:
Use `union a b` (Markdown notation) to make it print right in the documentation.